src/amo/components/Categories/index.js

```diff
@@ -51,6 +51,12 @@ export class CategoriesBase extends React.Component {
     const { addonType, className, error, loading, i18n } = this.props;
     const categories = this.props.categories[addonType] ?
       Object.values(this.props.categories[addonType]) : [];
+    categories.sort(function(a,b){
```

Comment:
@tofumatt   eslint test gives 58:14  error  Do not nest ternary expressions  no-nested-ternary
Should I change it to if else statements?